session/session.js

```diff
@@ -69,7 +69,11 @@ defineMethod(Session.prototype, 'resetMaxAge', function resetMaxAge() {
  */
 
 defineMethod(Session.prototype, 'save', function save(fn) {
-  this.req.sessionStore.set(this.id, this, fn || function(){});
+  if (this.req.sessionStore.set.length >= 4) {
```

Comment:
Creating a store does not require inheriting from this class. Please ensure your changes will not break for stores who either do not inherit from this class or who inherit from an older version of this class if the intention is to land this as a non breaking change.